lib/cs_guide_web/views/venue_view.ex

```diff
@@ -65,8 +65,4 @@ defmodule CsGuideWeb.VenueView do
     |> Enum.join("+")
   end
 
-  def order_drinks_by_type(drinks) do
```

Comment:
The drink are now sorted on the venue controller instead of the view

Reply:
@SimonLab makes sense. 👍